### PR TITLE
[O11y][Oracle] Add custom number formatting to visualisations

### LINKED ISSUE #7117: [O11y][Oracle] Add custom number formatting to visualizations
### Workaround implemented during lens migration
Used default format and migrated visualization to lens.
### Task
Revisit panels and add custom number formatting (2000->2k) to visualizations. 
[[Reference](https://github.com/elastic/integrations/pull/5830#pullrequestreview-1378314209)]
image
## Related Issues
- https://github.com/elastic/kibana/issues/72711
- https://github.com/elastic/integrations/issues/5753

### PR DESCRIPTION
**Pull request by kush-elastic**
### Workaround implemented during lens migration
Used default format and migrated visualization to the lens.

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Updated number formatting for the following dashboards

- [Metrics Oracle] Performance Metrics
- [Metrics Oracle] System Statistics Metrics
- [Metrics Oracle] Sysmetric Metrics
- [Metrics Oracle] Tablespace
- [Metrics Oracle] Overview

## Related issues

- Closes #7117 

## Note

According to this [comment](https://github.com/elastic/integrations/pull/5830#pullrequestreview-1378314209) received for MongoDB, we also checked for the panels in Oracle that require custom number formatting. However, it had an open [issue](https://github.com/elastic/kibana/issues/72711) which was released in the 8.10 version of Kibana. Hence updated the Kibana version to 8.10.2 for the change to be available in Kibana.

### CODE REVIEW

**Review by milan-elastic — APPROVED** [review]
LGTM!

**Review by agithomas — COMMENTED** [review]
Parse time value (bar chart) is showing negative values. Similarly others as well. Please revisit

**Review by agithomas — COMMENTED** [review]
Sysmetric dashboard is not consistant in terms of usage of theme colors. Please align the color themes

**Review by agithomas — COMMENTED** [review]
Ratio of used space dashboard is not showing any value. Please revist

**Review by agithomas — COMMENTED** [review]
Please align the Lock/Pin dashboard of Oracle Overview dashboard seems to be not aligned with the colors. Please revisit

**Comment by kush-elastic** [comment]
> Parse time value (bar chart) is showing negative values. Similarly others as well. Please revisit

Please refer to the previous [conversation](https://github.com/elastic/integrations/pull/5815#issuecomment-1506444750) for the same. This is applicable to all the visualizations that are using differences currently.

> Sysmetric dashboard is not consistant in terms of usage of theme colors. Please align the color themes
> Ratio of used space dashboard is not showing any value. Please revist
> Please align the Lock/Pin dashboard of Oracle Overview dashboard seems to be not aligned with the colors. Please revisit

Updated.

**Comment by agithomas** [comment]
Requesting for a revalidation. Please refer to the panel below. All the values are exactly the same to the 1/1000th place.  Requesting for a re-check if the viz formula are indeed correct.  Please confirm

image

**Comment by agithomas** [comment]
For other panels when Y axis value is percentage, `%` is added in the Y-axis values. Please add it here to ensure consistency.

**Comment by agithomas** [comment]
Please refer here . Is the Y-axis value '%' ? if Yes, please add that . 

image

**Comment by kush-elastic** [comment]
> [Requesting for a revalidation. Please refer to the panel below. All the values are exactly the same to the 1/1000th place. Requesting for a re-check if the viz formula are indeed correct. Please confirm](https://github.com/elastic/integrations/pull/8381#issuecomment-1825236101)

Checked the formula and it is as expected. I have updated the screenshot by populating the data so it shows different values and sorts them properly. Updated the visualization in Overview and Tablespace dashboard as it is present at both places. Also one more thing I noticed, as per this [comment](https://github.com/elastic/integrations/pull/8381#pullrequestreview-1731734279), the visualization was not showing values when it reaches the top limit of 10 values. Hence, increased the size of visualization as well to accumulate the values.

> [For other panels when Y axis value is percentage, % is added in the Y-axis values. Please add it here to ensure consistency.](https://github.com/elastic/integrations/pull/8381#issuecomment-1825237743)

Added the same

> [Please refer here . Is the Y-axis value '%' ? if Yes, please add that.](https://github.com/elastic/integrations/pull/8381#issuecomment-1825240894)

For this visualization, please refer to the documentation [here](https://docs.oracle.com/cd/B16240_01/doc/doc.102/e16282/oracle_database_help/oracle_database_instance_efficiency_cpuusage_ps.html#:~:text=Description,being%20performed%20by%20the%20database.). I don’t find it as a percentage field.


Also while looking at the Ratio of used space in data file visualization Tablespace dashboard, I found that the Rank By is not set to use proper fields, hence updated that as well. Please have a look at the settings before and after.

Before:-
image

After:-
image

File: packages/oracle/manifest.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+version: "1.23.0"
 description: Collect Oracle Audit Log, Performance metrics, Tablespace metrics, Sysmetrics metrics, System statistics metrics, memory metrics from Oracle database.
 type: integration
 categories:
   - observability
   - datastore
 conditions:
   kibana:
-    version: "^8.8.0"
+    version: "^8.10.2"
```
[the remark this answers is not in the document]
Comment (kush-elastic) [reply packages/oracle/manifest.yml:12]: Right, I have updated the PR description with a reason.

**Review by agithomas — APPROVED** [review]
LGTM!

**Pull request merged**